app/presenters/reservation_calendar.rb

```diff
@@ -1,15 +1,14 @@
 class ReservationCalendar < SimpleDelegator
 
-  attr_accessor :hostname, :ical, :protocol
+  attr_accessor :url, :ical
 
-  def self.to_calendar(reservation, hostname, protocol)
-    new(reservation, hostname, protocol).to_ical
+  def self.to_calendar(reservation, url = nil)
```

Comment:
Not a big deal as nothing is using this yet, but this seems like a good candidate for a keyword arg.